src/services/goToDefinition.ts

```diff
@@ -330,15 +335,11 @@ namespace ts.GoToDefinition {
 
     /** Returns a CallLikeExpression where `node` is the target being invoked. */
     function getAncestorCallLikeExpression(node: Node): CallLikeExpression | undefined {
-        const target = climbPastManyPropertyAccesses(node);
-        const callLike = target.parent;
+        const target = findAncestor(node, n => !isRightSideOfPropertyAccess(n));

```

Comment:
Is this related to the bug?

Reply:
Also a leftover when I was trying a different approach, and thought that it might be good to leave it.  (Same as above, I'll remove it if preferred.)